README.md

```diff
@@ -17,237 +17,178 @@
+There are commercially available RPC endpoints, however, we find them inadequate due to excessive cost and significantly slower speed of access. Erigon is easy to install, syncs to the tip of the chain very quickly, and is blazingly fast at serving data -- if one has an index. A great "devops" solution to running both Erigon and TrueBlocks is the [dAppNode project](https://github.com/dappnode).
 
-A docker build environment is also required.
+A [docker build environment](https://docs.docker.com/get-docker/) is also required to build this package.
 
-## Quick Start
+## Quick start
 
-When working properly, this docker package:
+When properly installed, `trueblocks-docker`:
 
 - builds `trublocks-core`,
 - runs `chifra init` to initialize the Unchained Index (this takes a while to complete),
-- runs `chifra scrape` (in the background) to actively build the index against the head of the chain,
-- runs `chifra serve` (in the background) to serve an API of all `chifra` commands,
-- optionally, runs `chifra monitors --watch` (in the background) to monitor a collection of addresses.
+- starts `chifra scrape` (in the background) to actively build the index against the head of the chain,
+- starts `chifra serve` (in the background) to serve an API of all `chifra` commands,
+- optionally, starts `chifra monitors --watch` (in the background) to monitor a collection of addresses.
 
 ### Getting started
 
-Make a backup copy of your `.env` file if you wish to preserve it, then:
+Copy the `env.example` file to `.env` and modify values (see the comments in the `env.example` for more details), or just do:
 
 ```bash
-echo "TB_CHAINS_MAINNET_RPCPROVIDER=your-RPC-url-and-port" >.env
+echo "TB_CHAINS_MAINNET_RPCPROVIDER=http://yourRpcEndpoint:port" >.env
 docker compose up
 ```
-You may replace `MAINNET` with `SEPOLIA` or `GNOSIS` in the above, as these chains are pre-configured.
 
-For other EVM-compatible chains, edit a file in the local folder called `.env` and add these items:
+Make sure to provide a valid RPC endpoint that exposes both an archive node and the `trace_` namespace. If you experience problems, you 
+may find useful answers in the comments of the `docker-compose.yml` and `docker-compose.local.example` files.
+
+### Running against other chains
+
+If you want to run against other EVM-compatible chains, edit a file in the local folder called `.env` (or copy the `env.example` file first) and add these items:
 
 ```bash
+TB_SETTINGS_DEFAULTCHAIN="<chain_name>"
 TB_CHAINS_`<CHAIN_NAME>`_CHAINID="<chain_id>"
 TB_CHAINS_`<CHAIN_NAME>`_RPCPROVIDER="<rpc endpoint>"
 TB_CHAINS_`<CHAIN_NAME>`_SYMBOL="<currency symbol>"
 ```
 
-then
+Of course, replace `<chain_name>` with (what else?) your chain's name.
 
-```
-docker compose up
-```
+Note: `MAINNET`, `SEPOLIA` and `GNOSIS` chains are pre-configured and you may use these values and the first two settings alone without further adue.
```

Comment:
It seems that "first two settings" point to "TB_SETTINGS_DEFAULTCHAIN" and "TB_CHAINS_`<CHAIN_NAME>`_CHAINID" and I'm not sure if this is what you wanted?

Reply:
Typo. Thanks.